source/help/settings/channel-settings.rst

```diff
@@ -21,11 +19,16 @@ notification preference for each channel, click the channel name at the
 top of the page to access the channel menu, then click
 **Notification Preferences > Send Desktop Notifications**.
 
+Send mobile push notifications
```

Comment:
Should we title case this? I know we don't in the product but I think that's okay for the docs